.github/bin/lint-whitespace

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+
+repo_root=$(git rev-parse --show-toplevel)
+cd "${repo_root}" || exit
```

Comment:
Rationale: the current `git grep` commands only search in the working directory. CI already runs the script from the repo root dir, but it's nice if a user doesn't have to.

Another approach is `git grep -- :/`, but then the paths are printed relative to the working directory, and I'd prefer them to be relative to the repo root.

If a user's working directory is a different repo (not configlet), this script will currently check that repo instead. If we wanted to avoid that, we can instead `cd` after getting the script's location.